dist/index.html

```diff
@@ -9,8 +9,7 @@
 
   <body>
     <h1>Safety Dungeon Master Control Centre</h1>
-    <div id="firebaseui-auth-container"></div>
```

Comment:
I assume these aspects are now handled in some other Vue component so they no longer need to be handled here